server.js

```diff
@@ -38,28 +36,117 @@ var renderHtml = function(url, cb) {
             }, 10000); //If no callPhantom script was called on the page, then wait 10secs and load page anyway.
         });
     };
+
+    console.debug("Opening page");
     page.open(url);
 
     page.onError = function (msg, trace) {
-        console.log(msg);
+        console.error(msg);
         trace.forEach(function(item) {
-            console.log('  ', item.file, ':', item.line);
+            console.error('  ', item.file, ':', item.line);
         });
     };
 };
 
+var configure = function() {
+    console.info("Reading config.json");
+
+    var content = fs.read('config.json');
+    var settings = JSON.parse(content);
+
+    var options = {}
+
+    if (fs.exists('/etc/prerender.json')) {
```

Comment:
The overwrite itself is there to keep settings, even when pulling from git. Without having merge conflicts etc..